ftw/globalstatusmessage/tests/test_controlpanel.py

```diff
@@ -153,3 +157,32 @@ def test_receiver(self, browser):
             },
             self.expected_settings
         )
+
+    @browsing
+    def test_receiver_filters_not_exitsing_paths_before_setting_theme(self, browser):
```

Comment:
Do you mind fixing the two typos in `exitsing` and `theme` because it's made grepping the source code harder.

Reply:
thanks and done

Reply:
Nice 👍